packages/ui/exchange-ui-core/src/containers/SupplyTable.tsx

```diff
@@ -0,0 +1,229 @@
+import React, { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { BigNumber } from 'ethers';
+import { Edit, Remove } from '@material-ui/icons';
+import {
+    getBackendClient,
+    formatCurrencyComplete,
+    moment,
+    usePermissions,
+    useTranslation,
+    EnergyFormatter,
+    IPaginatedLoaderHooksFetchDataParameters,
+    TableMaterial,
+    usePaginatedLoaderFiltered,
+    CustomFilterInputType,
+    ICustomFilterDefinition,
+    Requirements,
+    TableFallback,
+    checkRecordPassesFilters,
+    getCurrencies
+} from '@energyweb/origin-ui-core';
+import { UpdateSupplyModal, RemoveSupplyConfirmation } from '../components/modal';
+import { getEnvironment } from '../features/general';
+import { IDeviceWithSupply, ISupplyTableRecord } from '../types';
+import { getSupplies, fetchSupplies } from '../features/supply';
+
+export const KeyStatus = {
+    Active: 'Active',
+    Paused: 'Paused'
+};
```

Comment:
Probably better suited to be an `enum`. Also maybe better to rename it to `SupplyStatus` to make the intention clearer.

```
export enum SupplyStatus {
    Active = 'Active',
    Paused = 'Paused'
}
```